components/description-list/demo/description-list-test.js

```diff
@@ -42,6 +42,7 @@ class TestDescriptionList extends LitElement {
 	constructor() {
 		super();
 		this.type = 'default';
+		this.breakpoint = 200;
```

Comment:
Rather than explicitly defining this here (as well as in `d2l-dl-wrapper`), you could leave it without a default and use `ifDefined(this.breakpoint)` in the renderer.

Reply:
Oh yeah, whoops forgot about this! 😓